README.md

```diff
@@ -313,6 +313,7 @@ Table of Contents
   * [Unixtime](https://unixtime.co.za) - Free API to convert Unixtime to DateTime and vice versa.
   * [unscan](https://unscan.co) - Free API to scan images for NSFW and files for malware. 100 requests per second.
   * [Webhook.site](https://webhook.site) - Easily test HTTP webhooks with this handy tool that displays requests instantly.
+  * [Webhook.store](https://github.webhook.store) - Tool for storing third party webhooks and debug them on localhost (ngrok style). Open source and self-hostable. Free personal domain *username*.github.webhook.store, free public domains *anything*.webhook.store. 
```

Comment:
Right ! I heard from Heroku... I'll update my documentation to add other way to self host for free and ping you

Reply:
@ripienaar I have updated the readme documentation here https://www.openwebhook.io/docs/self-hosting/install-store-heroku .
Did you see it somewhere else ?

Reply:
Doc still says to sign up for free heroku, that wont be possible soon.

Reply:
Sorry 😬😬 I have now removed all mentions to any free tier from Heroku :)

Reply:
OK then last 2 suggestions is that you should make the url to `https://www.openwebhook.io` so people can see what this is about when they click on the link rather than just a github auth prompt which for most dont mean anything and so update the `webhook.store` part since thats not a valid url.